QuasarHelios2015/src/org/team1540/quasarhelios/AutoEjector.java

```diff
@@ -35,12 +35,14 @@ public static BooleanStatus create() {
     @Override
     protected void autonomousMain() throws AutonomousModeOverException, InterruptedException {
         try {
-            setClampHeight(1.0f);
-            Elevator.setBottom.event();
-
-            waitUntil(Elevator.atBottom);
-
             float currentClampHeight = Clamp.height.get();
+            setClampHeight(1.0f);
+            
+            if (!Elevator.atBottom.get()) {
+                Elevator.setBottom.event();
+                waitUntil(Elevator.atBottom);
+            }
+            
```

Comment:
There is also a little bit of unneeded whitespace here. It might just be me being picky.